IdentityCore/src/cache/MSIDToken.h

```diff
@@ -23,22 +23,25 @@
 
 #import <Foundation/Foundation.h>
 
-typedef NS_ENUM(uint32_t, MSIDTokenType)
+typedef NS_ENUM(NSInteger, MSIDTokenType)
 {
-    MSIDTokenTypeAccessToken =  'acTk',
-    MSIDTokenTypeRefreshToken = 'rfTk'
+    MSIDTokenTypeAccessToken,
+    MSIDTokenTypeRefreshToken
 };
 
-@interface MSIDToken : NSObject
+@interface MSIDToken : NSObject <NSSecureCoding>
 
 @property (readonly) NSString *token;
-
 @property (readonly) NSString *idToken;
 @property (readonly) NSDate *expiresOn;
 @property (readonly) NSString *familyId;
 @property (readonly) NSDictionary *clientInfo;
```

Comment:
Question for future: do we actually need to store the whole clientInfo? I'd suggest storing just userIdentifier

Reply:
"userIdentifier" is a combination of "uid" + "utid". If we store "userIdentifier" only, we should be ready to parse it back to "uid" & "utid".

Reply:
The idea is that the client shouldn't try to parse or use those values other than for user identification. And user identification is uid.utid